sagemaker-python-sdk/sparkml_serving_emr_mleap_abalone/sparkml_serving_emr_mleap_abalone.ipynb

```diff
@@ -21,7 +21,7 @@
     "## Setup an EMR cluster and connect a SageMaker notebook to the cluster\n",
```

Comment:
<p>Thanks for updating the serializers!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3033/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>Welcome. I have removed the individual JARs and used the <code>spark-shell</code> to resolve the dependencies. I had included JARs since I had not found the config to overwrite the default library, found that now!</p>